mu-plugins/central-hub/tests/phpunit/integration/meta-data/getCustomFieldsValues.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Tests for the function get_custom_fields_values().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Integration\Metadata;
+
+use function get_post_meta;
+use function add_post_meta;
+use function delete_post_meta;
+use function spiralWebDB\Metadata\get_custom_fields_values;
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+/**
+ * Class Tests_GetCustomFieldsValues
+ *
+ * @package spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @group   meta-data
+ */
+class Tests_GetCustomFieldsValues extends Test_Case {
+
+	/**
+	 * Instance of the post object for each test.
+	 *
+	 * @var WP_Post
+	 */
+	protected $post;
+
+	/**
+	 * Empty the store before starting these tests.
+	 */
+	public static function setUpBeforeClass() {
+		self::empty_the_store();
+	}
+
+	/**
+	 * Set up each test.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		// Create and get the post_id via the factory method.
+		$this->post = self::factory()->post->create();
+	}
+
+	/**
+	 * Clean up the test environment after each test.
+	 */
+	public function tearDown() {
```

Comment:
@rgadon107 This method isn't doing anything unique. Therefore, you can delete it.

Reply:
Done.